### PR TITLE
Tablet tab bar label colors

### PR DESCRIPTION
**Pull request by mlsteele**
Change ipad tab bar colors to match desktop labels.

image
image

### CODE REVIEW

File: shared/router-v2/router.native.tsx [hunk 1]
```diff
@@ -210,7 +210,13 @@ const VanillaTabNavigator = createBottomTabNavigator(
             <Kb.Text
               // @ts-ignore expecting a literal color, not a getter
               style={{
-                color: focused ? Styles.globalColors.whiteOrWhite : Styles.globalColors.blueDarkerOrBlack,
+                color: Styles.isDarkMode()
```
Comment (jakob223) [inline shared/router-v2/router.native.tsx]: I feel like this should be in a stylesheet

**Comment by mlsteele** [comment]
Uses stylesheet now. Still the same amount of switching for dark mode. I'm not sure when it's appropriate to add to global colors. Global colors has split values for dark mode but the names in global colors are all very generic. none like labelForThisParticularThing.

**Comment by jakob223** [comment]
I added globalColors for these 2 colors in the meantime for a different PR.

**Comment by mlsteele** [comment]
Almost, you added colors which share names but not dark-mode values. Think I should add `blueLighterOrWhite_50` and `whiteOrWhite_85`? Where neither `white_50` nor `white_85` exist.

**Comment by jakob223** [comment]
If you like. I would not consider myself an authority on the matter.

**Pull request merged**